src/components/dialog/faultEvent/FaultEventCreation.tsx

```diff
@@ -191,22 +191,18 @@ const FaultEventCreation = ({ useFormMethods, isRootEvent, eventValue, isEditedE
                 {...register("probability")}
               />
 
-              {(gateTypeWatch === GateType.PRIORITY_AND || !gateTypeWatch) &&
```

Comment:
Are u sure about removing all conditions? Because in my comment i mentioned only removal of `gateTypeWatch === GateType.PRIORITY_AND &&`.

To be honest i did not try to understand what is code doing, i just saw redundant line of code ... should i try to understand what it does? How come you can remove all conditions ?

Reply:
https://github.com/kbss-cvut/fta-fmea-ui/issues/406 is solved now